catroid/src/main/java/org/catrobat/catroid/stage/PreStageActivity.java

```diff
@@ -563,6 +595,31 @@ private void nfcInitialize() {
 		resourceInitialized();
 	}
 
+	private static ArrayList<Brick> getBricksRequieringResource(int resource) {
+		ArrayList<Brick> brickList = new ArrayList<Brick>();
+		ArrayList<Sprite> spriteList = (ArrayList<Sprite>) ProjectManager.getInstance().getCurrentProject()
+				.getSpriteListWithClones();
```

Comment:
Not really necessary to implement it in every container. Saw that the function `getRequiredResources` just uses the `sceneList` and iterates through their sprites. 
Did the same now.